dango/types/src/lending_pool.rs

```diff
@@ -0,0 +1,73 @@
+use grug::{Addr, Coins, Denom};
+
+/// The namespace that lending pool uses.
+pub const NAMESPACE: &str = "lending";
+
+#[grug::derive(Serde)]
+pub struct InstantiateMsg {
+    pub whitelisted_denoms: Vec<Denom>,
```

Comment:
Use `BTreeSet` over `Vec`
```suggestion
    pub whitelisted_denoms: BTreeSet<Denom>,
```

Reply:
Fixed